src/main/resources/mcmod.info

```diff
@@ -0,0 +1,16 @@
+[
+  {
+    "modid": "mwc",
+    "name": "Modern Warfare Cubed",
+    "description": "Modern Warfare Cubed, VMW: Forked. PMW",
```

Comment:
Please use the real description of this project:

> A well optimized, open source Modern Warfare rewrite, made in light of Vic's Modern Warfare!

Reply:
done